### PR TITLE
IOD-94 View Licensees by title

### PR DESCRIPTION
**Pull request by aideninbcgov**
# Description

This PR includes the following proposed change(s):

IOD-94

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [x] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by aideninbcgov** [comment]
view licensees done.  edit/remove licensees will be another ticket.

File: package-lock.json [hunk 1]
```diff
@@ -15,13 +16,15 @@
 				"@mui/styled-engine-sc": "5.4.2",
 				"@mui/styles": "5.5.0",
 				"@mui/x-data-grid": "5.8.0",
+				"@react-icons/all-files": "^4.1.0",
```
Comment (akroon3r) [inline package-lock.json:19]: carat @SDToews

File: package-lock.json [hunk 2]
```diff
@@ -15,13 +16,15 @@
...
 				"formik": "2.2.9",
 				"luxon": "2.3.1",
 				"next": "12.1.0",
 				"react": "17.0.2",
 				"react-date-picker": "8.4.0",
 				"react-datepicker": "4.7.0",
 				"react-dom": "17.0.2",
+				"react-icons": "^4.3.1",
```
Comment (akroon3r) [inline package-lock.json:27]: carat @SDToews

File: package.json [hunk 3]
```diff
@@ -17,6 +17,7 @@
 		"@mui/styled-engine-sc": "5.4.2",
 		"@mui/styles": "5.5.0",
 		"@mui/x-data-grid": "5.8.0",
+		"@react-icons/all-files": "^4.1.0",
```
Comment (akroon3r) [inline package.json:20]: carat @SDToews

File: src/components/common/ListMenu.jsx [hunk 4]
```diff
@@ -0,0 +1,83 @@
+// import { Avatar, Chip, Typography, Box, Menu, MenuItem, ListItemIcon , Logout } from '@mui/material';
```
Comment (akroon3r) [inline src/components/common/ListMenu.jsx:1]: do we need this line?

File: src/hooks/useSoftware/useSoftware.js [hunk 5]
```diff
@@ -20,5 +24,5 @@ export const useSoftware = (id) => {
         return softwareTable.create({ ...body, renewal })
     }
 
-    return { ...softwareTable, data, create, tableColumns, formFields }
-}
+    return { ...softwareTable, data, create, tableColumns, formFields, formColumns }
+}
```
Comment (akroon3r) [inline src/hooks/useSoftware/useSoftware.js:28]:
```suggestion
}

```

File: package.json (hunk 3 above)
Comment (SDToews) [reply package.json:20]: concur, remove the ^

File: src/components/formik/common/BaseControl.jsx [hunk 6]
```diff
@@ -14,6 +14,11 @@ export const BaseControl = (props) => {
 							*
 						</Typography>
 					) : null}
+					{availability ? (
```
Comment (SDToews) [inline src/components/formik/common/BaseControl.jsx:17]: I don't think this should be part of the BaseControl

File: src/components/formik/common/FormikControls.jsx [hunk 7]
```diff
@@ -25,6 +26,8 @@ export const FormikControls = (props) => {
 			return <DropdownFormikControl {...remainingProps} />;
 		case 'selectChip':
 			return <SelectChipFormikControl {...remainingProps} />;
+		case 'list':
```
Comment (SDToews) [inline src/components/formik/common/FormikControls.jsx:29]: let's use something like 'multiUserList' instead

File: src/components/table/TableContainer.jsx [hunk 8]
```diff
@@ -57,7 +57,6 @@ export const TableContainer = (props) => {
 				disableColumnMenu
 				disableSelectionOnClick
 				onRowClick={openItem}
-				// rowHeight={80}
```
Comment (SDToews) [inline src/components/table/TableContainer.jsx:60]: remove extraneous comments

**Review by SDToews — CHANGES_REQUESTED** [review]
a few inline comments to address.   Not keen on the MultiUserControl, let's get together and refactor

**Comment by aideninbcgov** [comment]
> a few inline comments to address. Not keen on the MultiUserControl, let's get together and refactor

Sounds good. Let's do that after the meeting.

**Review by SDToews — APPROVED** [review]

**Pull request merged**